docker/php/Dockerfile

```diff
@@ -32,3 +34,15 @@ RUN sed -ri -e 's!/var/www/html!${APACHE_DOCUMENT_ROOT}!g' /etc/apache2/apache2.
 COPY supervisord.conf /etc/supervisor/conf.d/supervisord.conf
 
 CMD ["/usr/bin/supervisord", "-n", "-c", "/etc/supervisor/conf.d/supervisord.conf"]
+
+# Always copy a file, but choose# Always copy a file, but choose xdebug.ini or an empty one xdebug.ini or an empty one
```

Comment:
This is a weird comment 😊

Reply:
Because 
1) COPY cannot be used inside an if, so I have to write always write the config file 
2) But if the config is there when xdebug is not installed, PHP will give an error because it tries to load a module that does not exists

So solutions was to overwrite the config file with an empty one, in case of no xdebug activated.